### DISCUSSION TITLE
Difference between latest and previous CoastalApp version

### DISCUSSION THREAD

**Discussion by sarriegi** [noaa-ocs-modeling/CoastalApp-testsuite#11] [Q&A] [answered]
Hello, my advisor asked me what are the main differences between the CoastalApp version that I got from Andre last spring and the latest CoastalApp version. Do you know which are these?

@pvelissariou1 @saeed-moghimi-noaa 

Angelos

**Comment by pvelissariou1** [comment, marked as answer]
Hi Angelos,

Regarding WW3, both versions of CoastalApp use Andre's scalability version
of WW3.
The recent versions of CoastalApp (main and develop branches) have
significant improvements
and bug fixes mostly related to the build system. Also I have updated
ADCIRC, PAHM and SCHISM to their latest versions.
The recent version of CoastalApp was tested using the CoastalApp-testsuite
(using, ADCIRC, ATMESH, PAHM, WW3) and all tests passed successfully.

I hope I answered your question.

Takis

Panagiotis Velissariou, Ph.D., P.E.
UCAR Scientist
National Ocean and Atmospheric Administration
National Ocean Service
Office of Coast Survey CSDL/CMMB
Physical Scientist - Project Lead
cell: (205) 227-9141
email: ***@***.***

**Reply by sarriegi** [reply]
Thank you very much @pvelissariou1 !
